### ISSUE TITLE
Release to the Maven central repo

### ISSUE THREAD

**Issue by highsource** [google/gcm#150]
When/if #149 is merged, could you please release to the central Maven repo? Many thanks.

**Comment by kroikie** [comment]
I am currently working on this. This will be made available very soon.

**Comment by maksimu** [comment]
Possible duplicate of #23

**Comment by highsource** [comment]
Great, thank you.

**Comment by highsource** [comment]
@kroikie Do you need any help with this? I'm blocked by this in a couple of issues, but don't want to deploy my fork the the central.

**Comment by kroikie** [comment]
The library is now available:

Group: com.google.gcm
Artifact: gcm-server
Version: 1.0.0

Thank you for your patience.

**Comment by highsource** [comment]
Great, many thanks.
